go.mod

```diff
@@ -76,7 +76,7 @@ require (
 	github.com/coredns/caddy v1.1.1 // indirect
 	github.com/coredns/corefile-migration v1.0.17 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
-	github.com/docker/distribution v2.8.1+incompatible // indirect
+	github.com/docker/distribution v2.8.2+incompatible // indirect
```

Comment:
This module is indirectly used by cluster-api package. Hi @sbueringer,  since no cluster-api bump in this PR, I assume you just update the github.com/docker/distribution v2.8.2+incompatible directly here, right?

Reply:
Yes, that is correct!

Reply:
Cool, thanks! I'll do the same then for capa.